content/blog/2020/release-0.2.0.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Release v0.2.0"
+description: "Announcing the v0.2.0 release of BDK"
+author: "Alekos Filini"
+date: "2020-12-21"
+tags: ["rust", "release"]
+hidden: true
+draft: false
+---
+
+A new release of BDK is finally out! The `v0.2.0` release contains many exciting new features, bug fixes and overall improvements. This release also marks the beginning of our new regular [release schedule][release_schedule], which will see us pushing
+out a new release every four weeks. We think this is a good compromise to ensure that developers using BDK have access to all the new features and fixes as soon as possible, at least while the library is still evolving very fast as it is
+right now. After `v1.0.0` we will increase this time to a more relaxed 6 weeks.
+
+You can find the full [v0.2.0 changelog][changelog] on GitHub.
+
+## What's new in v0.2.0
+
+Considering the sheer amount of new things being added. We don't have room here to explain every new feature in detail, but below is a quick overview of some you could find useful in your projects.
```

Comment:
I think you meant this to read: "... new things being added we don't have room here..."

Reply:
I remember fixing this by removing the period after "added". Basically it was "Considering the sheer amount of new things being added we don't have room here ..."

I might have made a mistake while committing the changes

Reply:
Forgot to push the amended commit :sweat_smile: